src/reducers/url-state.js

```diff
@@ -142,14 +151,14 @@ function transforms(state: TransformStacksPerThread = {}, action: Action) {
   switch (action.type) {
     case 'ADD_TRANSFORM_TO_STACK': {
       const { threadIndex, transform } = action;
-      const transforms = state[threadIndex] || [];
+      const transforms = state[threadIndex] || EMPTY_TRANSFORM_STACK;
```

Comment:
not useful because the array is sliced later on.

Only benefit is that no new object is allocated. I think this is a very small benefit for added complexity, but up to you.